Content.Client/Ghost/GhostSystem.cs

```diff
@@ -33,9 +32,10 @@ private bool GhostVisibility
 
                 _ghostVisibility = value;
 
-                foreach (var ghost in EntityQuery<GhostComponent, SpriteComponent>(true))
+                var query = AllEntityQuery<GhostComponent, SpriteComponent>();
+                while (query.MoveNext(out var uid, out _, out var sprite))
                 {
-                    ghost.Item2.Visible = true;
```

Comment:
This was changed to = true in this PR https://github.com/space-wizards/space-station-14/pull/9842 but I'm not sure why